### ISSUE TITLE
Contribution option

### ISSUE THREAD

**Issue by Snowmanko** [AlPepino/Next-Days#1]
Hi,
I would like to contribute on wiki side of addon and also mission code.
Do you think it will be possible to sent you some PR after you upload code here on git?

Thanks. BR!

**Comment by AlPepino** [comment]
Hi, what missions do you think of?

**Comment by Snowmanko** [comment]
Overall to share custom newly created missions and there is multiple infos missing in wiki how things work. I mean some description of missions, types and so on. 
etc.:
- how hunter/hunting skill works
- how clan tab works
- how tisi rocket mission works
- how radiation works, what items are contaminated in missions area
- how life insurance works and if it is needed to save stats
- how reputation works - what increases rep and what decreases

**Comment by AlPepino** [comment]
that sounds good. Even If I think almost none reads WiKi ...unfortunately.

**Comment by Snowmanko** [comment]
Yes, but server admins and owners will be able to use it in theirs webpages and server rules.

**Comment by Snowmanko** [comment]
Will you share it?

**Comment by AlPepino** [comment]
I will

**Comment by Snowmanko** [comment]
Great, I will be happy to contribute.
